graphql_compiler/compiler/filters.py

```diff
@@ -829,6 +829,11 @@ def _get_filter_op_name_and_values(directive):
     if 'op_name' not in args:
         raise AssertionError(u'op_name not found in filter directive arguments!'
                              u'Validation should have caught this: {}'.format(directive))
+    if args['op_name'].value.value in UNARY_FILTERS and 'value' not in args:
+        args['value'] = Argument(Name('value'), ListValue([], loc=args['op_name'].loc))
+    if args['op_name'].value.value not in UNARY_FILTERS and 'value' not in args:
+        raise GraphQLValidationError(u'Directive value omitted for'
```

Comment:
nit: we've adopted the convention of leaving the whitespace before the line break, rather than after it